### ISSUE TITLE
Error "redirect_uri_mismatch" on backend

### ISSUE THREAD

**Issue by norman74** [EddyVerbruggen/cordova-plugin-googleplus#619]
First, I should say that using the same OAuth client ID works perfectly for Web application but fails when i try to use it with this plugin. 

Also, **client successfully generates serverAuthCode** which then sends to the server, but when server need to exchange this code for an accesToken, it returns **redirect_uri_mismatch**. 

As I said before it works fine when **serverAuthCode** is sent from web application.

I use "postmessage" for redirect uri since non of others doesn't seem to work and there's no need for client redirecting.

**Comment by wannesmatthys** [comment]
Any chance you found the fix? Running in the same problem.

**Comment by elfwy-uwu** [comment]
> Any chance you found the fix? Running in the same problem.

Yeah, i think i've fixed this somehow. I don't unfortunately remember a thing about this. You can try switching to another plugin. Seems like this one is not supported anymore since google+ was shut down 2 years ago
